### DISCUSSION TITLE
[VSCode] Support per workspace configuration

### DISCUSSION THREAD

**Discussion by rrebollo** [odoo/odoo-ls#113] [Ideas]
Wouldn't be nice if extension configuration could be manage at workspace level? I'm talking about this set of configurations:
![image](https://github.com/odoo/odoo-ls/assets/38138332/70f0784a-1b4f-4d5b-8cbe-01392cb3abfa)

**Comment by fda-odoo** [comment]
Hello,

Thank you for your feedback. If I understand well, you would like that you could put a path to your odoo and addons related to the current workspace path.
This is a feedback we already got, and this improvement should come in the next update !
If I misunderstood your issue, do not hesitate to answer to this thread, I will close it otherwise
See https://github.com/odoo/odoo-ls/issues/90

**Reply by rrebollo** [reply]
Your're right, U can close this. Sorry.
